Update README.md (#391)
leikoilja/ha-google-home: README.md

   - [Manual Installation](#manual-installation)
   - [Integration Setup](#integration-setup)
   - [Running in Home Assistant Docker container](#running-in-home-assistant-docker-container)

>    - [ARM Docker Container Dependencies (Workaround)](#arm-docker-container-dependencies-workaround)

Reviewer: ````suggestion
   - [Installing ARM Docker Container Dependencies (Workaround)](#arm-docker-container-dependencies-workaround)
```

Make sure that you have your Home Assistant Container network set to `host`, as perscribed in the official docker installation for Home Assistant.

### ARM Docker Container Dependencies Workaround

> If on installing the integration on an ARM based Home Assistant container (Raspberry Pi, Synology, etc), an error occurs: 

Reviewer: ```suggestion
If you are installing this integration on an ARM based device (Like Raspberry Pi, Synology, etc), you may need to do the following if you get this error:
```

```
ERROR: Cannot install glocaltokens==0.3.1
```

> This can be installed manually by accessing the commandline of the HA Docker container and running:

Reviewer: ```suggestion
Please run the following command in the Home Assistant container to add the missing dependencies for `glocaltokens`:
```

```

> apk add gcc

Reviewer: ```suggestion
apk add gcc g++
```

`build-essential` may also be need.

> apk add g++

Reviewer: ```suggestion

```

Reviewer: Please remove this line

```

> then

Reviewer: ```suggestion
then you can install `glocaltokens` manual like this:
```

```
pip3 install glocaltokens
```

This will need to be done each time the image is updated unfortunately. Alternatively you can add the command to the container startup:

```

> command: /bin/bash -c "apk add gcc g++; pip3 install --upgrade pip; pip3 install pymysql glocaltokens==0.3.1; /init"

Reviewer: Why `pymysql`?

```suggestion
command: /bin/bash -c "apk add gcc g++; pip3 install --upgrade pip; pip3 install glocaltokens==0.3.1; /init"
```

Make sure that you have your Home Assistant Container network set to `host`, as perscribed in the official docker installation for Home Assistant.

### ARM Docker Container Dependencies Workaround

If you are installing this integration on an ARM based device (Like Raspberry Pi, Synology, etc), you may need to do the following if you get this error:

```

> ERROR: Cannot install glocaltokens==0.3.1

Reviewer 2: ```suggestion
ERROR: Cannot install glocaltokens==<version>
```

```
Please run the following command in the Home Assistant container to add the missing dependencies for `glocaltokens`:

```
apk add gcc g++
```
then you can install `glocaltokens` manually like this:
```
pip3 install glocaltokens
```

> This will need to be done each time the image is updated unfortunately. Alternatively you can add the command to the container startup:

Reviewer 2: ```suggestion
Unfortunately, this will need to be done each time the image is updated. Alternatively you can add the command to the container startup:
```

Make sure that you have your Home Assistant Container network set to `host`, as perscribed in the official docker installation for Home Assistant.

### ARM Docker Container Dependencies Workaround

If you are installing this integration on an ARM based device (Like Raspberry Pi, Synology, etc), you may need to do the following if you get this error:

```
ERROR: Cannot install glocaltokens==0.3.1
```
Please run the following command in the Home Assistant container to add the missing dependencies for `glocaltokens`:

```
apk add gcc g++

> apk add g++

Reviewer: ```suggestion

```
